Classes/Command/BaseCommand.php

```diff
@@ -196,7 +194,7 @@ protected function saveToDatabase(Document $document)
         $document->setAuthor(implode('; ', $metadata['author']));
         $document->setThumbnail($doc->thumbnail ? : '');
         $document->setMetsLabel($metadata['mets_label'][0] ? : '');
-        $document->setMetsOrderlabel($metadata['mets_orderlabel'][0] ? : '');
+        $document->setMetsOrderlabel($metadata['mets_orderlabel'][0] ? : $metadata['mets_order'][0] ? : '');
```

Comment:
The idea was to keep the document instance as close to the METS standard as possible in order to allow for future applications where there is a difference between having an `ORDERLABEL` or it being empty/omitted. If you need to have a fallback (i. e. `ORDER`) then this should be done when applying the property in the respective context.

But you are right, maybe we should consolidate this and just use a single field for sorting.

Nevertheless, since the change in this line seems to be unnecessary to achieve the goal of fixing the display of volumes, I'd rather keep it unchanged for now until we do a complete refactoring of the sorting mechanisms (like you suggested). Otherwise this could confuse people even more.

```suggestion
        $document->setMetsOrderlabel($metadata['mets_orderlabel'][0] ? : '');
```

Reply:
Nevermind, I just saw that the document object doesn't have a property which holds the `ORDER` value so using this as a fallback wouldn't be that easy.

So, please ignore my comment! I'll approve the pull request and merge it!